sdk/tables/azure-data-tables/README.md

```diff
@@ -4,15 +4,10 @@ key/attribute store with a schemaless design. Tables storage gives developers fl
 best parts of Azure cloud.
 
 [Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation]
-| [Product documentation][azconfig_docs] | [Samples][samples]
+| [REST API documentation][rest_api] | [Product documentation][product_documentation] | [Samples][samples]
```

Comment:
This is the only library that I know has a link to the REST API. is this something we're adding now?

Reply:
I was following along with what the Storage libraries did, but I see that more recent READMEs don't have this. Fixed.